### ISSUE TITLE
translate title

### ISSUE THREAD

**Issue by Tealk** [tommilligan/mdbook-admonish#168]
hello,

is it possible to translate the titles?

**Comment by tommilligan** [comment]
You can specify a [custom title](https://tommilligan.github.io/mdbook-admonish/#custom-title) for each block, if that would help.

The default titles aren't currently localized, I think this would be reasonably easy to add as a `book.toml` configuration option, if you want to work on it I'm open to PRs

**Comment by Tealk** [comment]
Yes, but then I have to do it for every block, it's not quite as nice as central.

The `book.toml` is a good idea, but I don't really know how the whole system works here. So I don't know if I can contribute to the code. As far as I have noticed, the CSS file is not sufficient to use mdbook-admonish.

**Comment by tommilligan** [comment]
The point where a directive (the thing like `note` that tells us which type of box to render) is converted to a title is here: https://github.com/tommilligan/mdbook-admonish/blob/8a0ecc5dd17495f53e3ec9687e2d35494d7d1f27/src/resolve.rs#L48

Because the number of directives is pretty limited, I wouldn't overthink this, I'd be happy with a big old hardcoded static list of translations by language. We can then decide which set of translations to use by loading a config option from the file, and threading it down like we do with custom directives here: https://github.com/tommilligan/mdbook-admonish/blob/8a0ecc5dd17495f53e3ec9687e2d35494d7d1f27/src/resolve.rs#L74

It may be the case using an internationalization framework like https://github.com/longbridgeapp/rust-i18n would make it simpler, but it might also make it more complex, kind of up to you which is easier.

The point I would add the config option for `book.toml` is here: https://github.com/tommilligan/mdbook-admonish/blob/8a0ecc5dd17495f53e3ec9687e2d35494d7d1f27/src/types.rs#L9

**Comment by Tealk** [comment]
We could use the `language = "de"` in the `book.toml`.
